shell/platform/embedder/embedder.h

```diff
@@ -2119,6 +2121,7 @@ typedef struct {
   FlutterEngineTraceEventDurationBeginFnPtr TraceEventDurationBegin;
   FlutterEngineTraceEventDurationEndFnPtr TraceEventDurationEnd;
   FlutterEngineTraceEventInstantFnPtr TraceEventInstant;
+  FlutterEnginePostPlatformThreadTaskFnPtr PostPlatformThreadTask;
```

Comment:
There should be a function version of this API as well; this struct is intended as a 1:1 alternative to the functions above.